docs/configuration.md

```diff
@@ -1664,6 +1664,15 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.port</code></td>
+        <td>0</td>
+        <td>
+            Port number to listen on for SSL connections. 
+            Default value of 0 means the port will be determined automatically.
+            Attention that the port should be separated for each particular protocols.
```

Comment:
I think it would be clearer to write something like "Note that the SSL port for individual services can be overridden by setting values like `spark.ssl.YYY.port` as described above."